swarm/network/simulation/kademlia_test.go

```diff
@@ -43,16 +59,22 @@ func TestWaitTillHealthy(t *testing.T) {
 			b.Store(BucketKeyKademlia, kad)
 			return network.NewBzz(config, kad, nil, nil, nil), nil, nil
 		},
-	})
+	}
+
+	// create the first simulation
+	sim := New(simServiceMap)
```

Comment:
instead of `defer sim.Close()`, we should have `sim.Close()` on line 95, no?

Reply:
yes indeed

Reply:
?